src/Repository/AdherentEmailSubscriptionHistoryRepository.php

```diff
@@ -0,0 +1,75 @@
+<?php
+
+namespace AppBundle\Repository;
+
+use AppBundle\Entity\Adherent;
+use AppBundle\Entity\AdherentEmailSubscriptionHistory;
+use AppBundle\Entity\ReferentTag;
+use Doctrine\Bundle\DoctrineBundle\Repository\ServiceEntityRepository;
+use Symfony\Bridge\Doctrine\ManagerRegistry;
+
+class AdherentEmailSubscriptionHistoryRepository extends ServiceEntityRepository
+{
+    public function __construct(ManagerRegistry $registry)
+    {
+        parent::__construct($registry, AdherentEmailSubscriptionHistory::class);
+    }
+
+    /**
+     * @param Adherent $adherent
+     *
+     * @return AdherentEmailSubscriptionHistory[]
+     */
+    public function findByAdherent(Adherent $adherent, $withoutInactives = true): array
```

Comment:
where do you use the second argument (outside tests)? if not used, IMO not need to add this only for test.